server/lib/src/constants/schema.rs

```diff
@@ -211,6 +211,40 @@ pub const JSON_SCHEMA_ATTR_NAME_HISTORY: &str = r#"{
     }
 }"#;
 
+pub const JSON_SCHEMA_ATTR_EC_KEY: &str = r#"{
+    "attrs": {
+      "class": [
+        "object",
+        "system",
+        "attributetype"
+      ],
+      "description": [
+        "Account verification public and optionally private key."
+      ],
+      "index": [
+        "EQUALITY"
+      ],
+      "unique": [
+        "true"
```

Comment:
don't need unique here, I think it just adds cost to the update of these values, and it requires indexing.